Worker.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
   <metadata>
     <id>Microsoft.Azure.Functions.NodeJsWorker</id>
-    <version>1.0.0-beta1-$version$</version>
+    <version>1.0.0-beta1$version$</version>

```

Comment:
- is appended to the version in appveyor.yml. So for dev builds nuget packages would look like 1.0.0-beta1-buildnumber and for tagged releases, it would match the version specified in the package.json